lib/capistrano/tasks/rails_artifact.rake

```diff
@@ -1,26 +0,0 @@
-namespace :rails_artifact do
-  desc 'Check that the repo is reachable'
-  task :check do
-  end
-
-  desc 'Copy repo to releases'
-  task :create_release do
-    on release_roles :all do
```

Comment:
Is there anything you need to know regarding permissions? Should it be added to the Readme?